tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -361,13 +374,14 @@ int Main(int argc, char** argv) {
         {"max_profiling_buffer_entries", required_argument, nullptr, 'e'},
         {"warmup_runs", required_argument, nullptr, 'w'},
         {"gl_backend", required_argument, nullptr, 'g'},
+        {"hexagon_delegate", required_argument, nullptr, 'j'},
```

Comment:
This is should be optional ? Why required argument ?

Reply:
required_argument means the optional requires an argument  (`-j 0|1` instead of `-j`), just like what nnapi (`-a 0|1`) and gpu (`-g 0|1`) delegates do.